tracing/README.md

```diff
@@ -0,0 +1 @@
+TODO
```

Comment:
Please file an issue to update the readme text for tracing and tracing/opencensus

Reply:
Added the structure to make it pass check-in, tracked in #3708